keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -200,6 +227,34 @@ Spec.Containers[i].ResourcesAllocated are used to determine if there is enough
 room to admit the Pod. Kubelet does not set Pod's ResourcesAllocated after
 admitting a new Pod.
 
+<<[UNRESOLVED]>>
+The above is confusing.  Does it mean that ResourcesAllocated will be set by
+someone other than Kubelet?  Who?  Why?
+
+Above you said 'limit access to ResourcesAllocated field such that only Kubelet
+can update this field' which is the opposite of 'kubelet does not set Pod's
+ResourcesAllocated after admitting' here.  I am very confused.
+
+Naively, I would expect it to be:
+
+Create:
+* user creates a pod with `resources`
+* scheduler schedules based on new pod `resources` and existing pods'
+  `allocated`
+* kubelet accepts the pod based on new pod `resources` and existing pods'
+  `allocated`
+* kubelet sets `allocated` = `resources` immediately (acts as an ACK)
```

Comment:
this step can be skipped as described above, and save a trip to apiserver and avoid running into api rate-limiting issues.

Reply:
to clarify, this step can be skipped because it is not required as described [above](https://github.com/kubernetes/enhancements/pull/1883/files#r449783195).

api rate limiting is just a potential issue we _might_ hit (no perf data to back this up at this time - btw, I looked into kubemark but don't know if there's a test I can specifically run to get some numbers on this) if we did this. sorry for causing confusion here.